ui/js/components/perf/compare.js

```diff
@@ -108,3 +108,52 @@ treeherder.component('compareError', {
         newRevision: '<'
     }
 });
+
+treeherder.component('distributionGraph', {
+    template: '<canvas id="a" width="400" height="30"></canvas>',
+    bindings: {
+        replicates: '<'
+    },
+    controller: [function () {
+        var ctrl = this;
+        var cvs = document.getElementById("a");
+        var ctx = cvs.getContext("2d");
+        var max = Math.ceil(Math.max.apply(null, ctrl.replicates)*1.01);
+        var min = Math.floor(Math.min.apply(null, ctrl.replicates)/1.01);
```

Comment:
Math.max(value) works only for single elements. As we are dealing with lists over here, I used max.apply.
I took up this logic from here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max